src/components/toggle-switch/toggle-switch.js

```diff
@@ -1,54 +1,67 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import styles from './toggle-switch.css';
 
 const cx = classNames.bind(styles);
 
-export default class ToggleSwitch extends Component {
+export default class ToggleSwitch extends React.Component {
```

Comment:
Any reason for this change? The previous way seems preferable.

Reply:
That somehow got in there in the Rebase...good catch!